readme.md

```diff
@@ -250,6 +250,7 @@
 - [Pyramid](https://github.com/uralbash/awesome-pyramid#readme)
 - [Play1 Framework](https://github.com/PerfectCarl/awesome-play1#readme)
 - [CakePHP](https://github.com/friendsofcake/awesome-cakephp#readme) - PHP framework.
+- [Swoft](https://github.com/wujunze/awesome-swoft#readme) -  Modern High performance AOP and Coroutine PHP Framework, base on Swoole.
```

Comment:
As per https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md 
```
The entry in the Awesome list should:
- Be added at the bottom of the appropriate category.
````
I understand that in **Back-End Development** we have various technologies and you probably wanted to add it in the end of the PHP frameworks, so probably it should be after **Lumen**, but per rules it should be in the end of the section after **Vapor**